bdai_ros2_wrappers/test/test_action.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2024 Boston Dynamics AI Institute Inc.  All rights reserved.
+import array
+import collections
```

Comment:
also here
```suggestion
from collections import deque
```

Reply:
Done in https://github.com/bdaiinstitute/ros_utilities/pull/58/commits/329efc3935f1f9fdc187d6157e4d218f4289adfd.